src/verifiers/documentStatus/didSignedDocumentStatus/didSignedDocumentStatus.ts

```diff
@@ -20,18 +23,152 @@ const skip: VerifierType["skip"] = async () => {
 };
 
 const test: VerifierType["test"] = (_document) => {
-  return true;
+  const document = _document as any; // TODO Casting to any first to prevent change at the OA level
```

Comment:
I dont understand this cast ?

Reply:
New schema does not work with the OA type. I dont want to deal with twiddling oa package until we settle what we like for the schema.